# Correction offset applied to object position [m]
object_position_correction_x: -0.045
object_position_correction_y: 0.02
object_position_correction_z: -0.007

# Real picking object manager settings
# Currently supported "bottle" and "cardbox_object"
object_type: bottle
gripper_joint_name: right_gripper

# Almost maximum, if needed can go to 1.2
opened_gripper_value: 1.0

bottle_closed_gripper_value: 0.2
cardbox_object_closed_gripper_value: 0.1

# Bottle size
bottle_cap_height: 0.02
bottle_cap_radius: 0.01

# Lower than real height - otherwise it colides with table scene box and plan
# to pick up bottle isn't good
bottle_height: 0.1
bottle_radius: 0.03
